configs/machines/blogin.yaml

```diff
@@ -11,9 +11,27 @@ choose_use_hyperthreading:
         True:
                 hyperthreading_flag: ""
         "0":
```

Comment:
I would use False here for consistency. May confuse new users who don't know that we treat 0 and false the same

Reply:
Also in the lines above we have 1 and True which seems a duplication which seems left from times where we weren't sure if 1 and true really do the same...

Reply:
The "duplication" here was added one year ago for safety to avoid problems that users have been experiencing (I think it was something in the lines that someone wrote `use_hyperthreading: '"0"'` and then this was a string and evaluated as true. Therefore, I would not remove yet this safety feature until the background problem is resolved.

Reply:
If you are okay with that then let me know and I can go ahead and merge :)